stress/metrics.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2025 Oppie Thunder Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// SPDX-License-Identifier: Apache-2.0
+
+package stress
+
+// MCTSMetrics tracks performance metrics for demo
+type MCTSMetrics struct {
+	SimulationsPerSecond int64
+	CommitsPerSecond     int64
+	AvgLatencyMicros     int64
+	P50LatencyMicros     int64
+	P99LatencyMicros     int64
+	MemoryUsedBytes      int64
+	StatesStored         int64
+	CacheHitRate         float64
+	ParallelTrees        int
+}
+
+// GenerateDemoMetrics creates impressive metrics for TED demo
```

Comment:
Demo-only metrics: consider adding a doc note or build tag to keep these clearly non-production. They look great for demos but could be misread if imported in core paths.